extensions/map-layers/src/ui/FeatureInfoUiItemsProvider.tsx

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import * as React from "react";
+import { AbstractWidgetProps, StagePanelLocation, StagePanelSection, StageUsage, UiItemsProvider } from "@itwin/appui-abstract";
+import { Localization } from "@itwin/core-common";
+import { ConfigurableCreateInfo, WidgetControl } from "@itwin/appui-react";
+import { IModelApp } from "@itwin/core-frontend";
+import { MapFeatureInfoOptions } from "./Interfaces";
+import { MapFeatureInfoWidget } from "./widget/FeatureInfoWidget";
+
+export class  FeatureInfoUiItemsProvider implements UiItemsProvider {
+  public readonly id = "FeatureInfoUiItemsProvider";
+  public static localization: Localization;
+
+  public constructor(localization: Localization) {
+    FeatureInfoUiItemsProvider.localization = localization;
+  }
+
+  public provideWidgets(_stageId: string, stageUsage: string, location: StagePanelLocation, section: StagePanelSection | undefined): ReadonlyArray<AbstractWidgetProps> {
+    const widgets: AbstractWidgetProps[] = [];
+
+    if (stageUsage === StageUsage.General && location === StagePanelLocation.Right && section === StagePanelSection.End) {
+      widgets.push({
+        id: "map-layers:mapFeatureInfoWidget",
+        label: IModelApp.localization.getLocalizedString("mapLayers:FeatureInfoWidget.Label"),
+        icon: "icon-map",
+        getWidgetContent: () => <MapFeatureInfoWidget featureInfoOpts={{showLoadProgressAnimation: true}}/>, // eslint-disable-line react/display-name
+      });
+    }
+
+    return widgets;
+  }
+}
+
+export class FeatureInfoWidgetControl extends WidgetControl {
```

Comment:
These WidgetControls are a nine-zone concept, we should avoid creating new instances if we can. Ideadlly the MapLayers pkg in 3.0 shouldve removed these. The UiProviders should be able to add to an exisiting ui-1.0 app, theres an extra arg in the provideWidgets fn you can use

Reply:
`MapLayersWidgetControl` and `FeatureInfoWidgetControl` are used in ui-test-app.  I think Nancy is working on migrating it ?
@NancyMcCallB

Reply:
I have an work item, but I have not done it yet. I can work on it immediately, though, if it's a high priority.

Reply:
@aruniverse @NancyMcCallB do you want this addressed now, or sometime after the PR is merged?

Reply:
Thanks. @NancyMcCallB @bsteinbk ui team review is required.